src/features/licenses/components/LicenseTable/columns.jsx

```diff
@@ -16,14 +20,31 @@ export const getColumns = ({ handleShowDetails, handleEditModal }) => [
     disableFilters: true,
   },
   {
-    Header: 'Master Courses',
-    accessor: ({ courses }) => (
-      <ul style={{ listStyleType: 'none', paddingLeft: 0 }}>
-        {courses.map(course => <li key={course}>{`${course.id} - ${course.displayName}`}</li>)}
-      </ul>
-    ),
+    Header: 'Master Courses / Catalogs',
+    accessor: 'courses',
     filter: 'text',
     disableSortBy: true,
+    Cell: ({ row }) => {
+      const { licenseType, courses, catalogs } = row.original;
+      const catalogsList = useSelector(state => state.licenses.catalogs.data);
+
+      if (licenseType === LicenseTypes.CATALOG) {
+        const elements = catalogs.map(catalog => filter(catalogsList, { value: catalog }));
+        return (
+          <ul style={{ listStyleType: 'none', paddingLeft: 0 }}>
```

Comment:
What about to use https://paragon-openedx.netlify.app/components/chip/ to make a clear distintion between courses and licenses?

Reply:
@01001110J I used badge instead of chip because is not necessary the interactivity of chip